core/storage/question/gae_models.py

```diff
@@ -244,6 +244,60 @@ def get_question_skill_links_and_skill_descriptions(
         )
         return question_skill_link_models, skill_descriptions, next_cursor_str
 
+    @classmethod
+    def get_all_question_ids_and_skill_descriptions(
+            cls, question_count, skill_ids, start_cursor):
+        """Fetches question ids and skill descriptions linked to a list of
+        skill ids.
+
+        Args:
+            question_count: int. The number of questions to be returned.
```

Comment:
Is this the right comment? We aren't actually returning question_count number of questions right? Its more like question_count number of question skill links?

Reply:
Also we aren't getting all anymore, maybe its better to write get_question_ids_and_skill_descriptions

Reply:
Done.